rmw_connextdds_common/include/rmw_connextdds/static_config.hpp

```diff
@@ -53,6 +53,11 @@
 #define RMW_CONNEXT_ENV_LOCALHOST_ONLY     "ROS_LOCALHOST_ONLY"
 #endif /* RMW_CONNEXT_ENV_LOCALHOST_ONLY */
 
+#ifndef RMW_CONNEXT_ENV_OVERRIDE_ENDPOINT_QOS_POLICY
+#define RMW_CONNEXT_ENV_OVERRIDE_ENDPOINT_QOS_POLICY     "RMW_CONNEXT_OVERRIDE_ENDPOINT_QOS_POLICY"
+#endif /* RMW_CONNEXT_ENV_ALLOW_TOPIC_QOS_PROFILES */
```

Comment:
change to `RMW_CONNEXT_ENV_ENDPOINT_QOS_OVERRIDE_POLICY` and `"RMW_CONNEXT_ENDPOINT_QOS_OVERRIDE_POLICY"`

Reply:
See https://github.com/rticommunity/rmw_connextdds/pull/7/commits/7e5fbe48e853c4708acbcf2e2d36f88f1f0f8cc9